core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1662,4 +1670,35 @@ public String toString() {
                 clientAddressSources, clientAddressTrustedProxyFilter, clientAddressFilter, clientAddressMapper,
                 enableServerHeader, enableDateHeader);
     }
+
+    public final ServerConfig buildServerConfig(ServerConfig existingConfig) {
```

Comment:
Ditto - This method should not be public but package-private.

Reply:
Yup.